lib/modules/dosomething/dosomething_api/resources/signup_resource.inc

```diff
@@ -189,7 +191,120 @@ function _signup_resource_index($user, $users, $campaigns, $count, $page, $compe
     'runs' => $runs,
   ];
 
-  return (new SignupTransformer)->index($parameters);
+  if (variable_get('rogue_collection', FALSE)) {
+    if ($user) {
+      $northstar_user_id = dosomething_user_get_northstar_id($user);
+      $user_object = user_load($user);
+    }
+
+     $rogue = dosomething_rogue_client();
+
+     $response = $rogue->getActivity([
+       'filter' => [
+         'northstar_id' => $northstar_user_id,
+         'campaign_id' => $campaigns,
+       ],
+     ]);
+
+    if ($response) {
```

Comment:
oooh this is so cool! I'm probably not understanding this correctly but I'm not sure if we can use this here because we want everything nested, not just parts returned?